packages/kbn-monaco/src/esql/lib/ast/validation/esql_validation_meta_tests.json

```diff
@@ -559,11 +559,11 @@
       "error": true
     },
     {
-      "query": "row var = coalesce(\"a\")",
+      "query": "row var = coalesce(\"a\", \"a\")",
```

Comment:
These all changed because these function definitions are now subject to [this `minParamsToAdd` logic](https://github.com/elastic/kibana/pull/177628/files#diff-b060292210212a042eee704ed17d6ff7fa2e5dbfac9c18252a0567d00dc37500R18). Not sure if this is okay(?)

Reply:
That look strange. In theory it should use the `minParams` amount of parameters for the test, but here I see `minParams + 1` used.
i.e. `coalasce("a")` should be the minimal signature in this case. 🤔

Reply:
Yeah, but `extraArg` currently [doesn't take no for an answer](https://github.com/elastic/kibana/pull/177628/files#diff-b060292210212a042eee704ed17d6ff7fa2e5dbfac9c18252a0567d00dc37500L20) :)

```ts
minParamsToAdd || 1
```